music21/test/testStream.py

```diff
@@ -5630,13 +5627,13 @@ def testVoicesToPartsA(self):
         s1 = s0.voicesToParts()
         self.assertEqual(len(s1.parts), 3)
         # self.assertEqual(len(s1.parts[0].flat), len(v1.flat))
-        self.assertEqual([e for e in s1.parts[0].flat], [e for e in v1.flat])
+        self.assertEqual(list(s1.parts[0].flat), list(e for e in v1.flat))
```

Comment:
second half can be `list(v1.flat)` otherwise it's an unnecessary generator expression